tests/bluetooth/bsim_bt/bsim_test_mesh/src/test_advertiser.c

```diff
@@ -262,6 +381,86 @@ static void test_tx_cb_multi(void)
 	PASS();
 }
 
+static void test_tx_proxy_mixin(void)
+{
+	static struct bt_mesh_prov prov = {
+		.uuid = test_prov_uuid,
+	};
+	int err;
+
+	/* Initialize mesh stack and enable pb gatt bearer to emit beacons. */
+	bt_mesh_device_setup(&prov, &comp);
+	err = bt_mesh_prov_enable(BT_MESH_PROV_GATT);
+	ASSERT_OK(err, "Failed to enable GATT provisioner");
+
+	/* Start pb gatt beacons with 100 msec interval */
+	k_sleep(K_MSEC(1800));
+
+	LOG_INF("Provision device under test");
+	/* Provision dut and start gatt proxy beacons. */
+	bt_mesh_provision(test_net_key, 0, 0, 0, adv_cfg.addr, adv_cfg.dev_key);
+
+	/* Start proxy beacons with 1 sec interval */
+	k_sleep(K_MSEC(6000));
+
+	/* Send adv data */
+	struct net_buf *buf = bt_mesh_adv_create(BT_MESH_ADV_DATA, BT_MESH_LOCAL_ADV,
+			BT_MESH_TRANSMIT(5, 20), K_NO_WAIT);
+	net_buf_add_mem(buf, txt_msg, sizeof(txt_msg));
+	bt_mesh_adv_send(buf, NULL, NULL);
+	k_sleep(K_MSEC(150));
+
+	/* Start proxy beacons again */
+	k_sleep(K_MSEC(6000));
+
+	PASS();
+}
+
+static void test_rx_proxy_mixin(void)
+{
+	static struct bt_mesh_test_gatt gattparam = {
+		.transmits = 1500 / 100,
+		.interval = 100,
+		.tolerance = 20,
+		.service = MESH_PROV_SERVICE
+	};
+
+	bt_init();
+
+	/* Scan pb gatt beacons. */
+	gatt_param = &gattparam;
+	rx_gatt_beacons();
+
+	/* Delay to provision dut */
+	k_sleep(K_MSEC(1000));
+
+	/* Scan proxy beacons. */
+	gattparam.transmits = 5000 / 1000;
+	gattparam.interval = 1000;
+	gattparam.tolerance = 100;
+	gattparam.service = MESH_PROXY_SERVICE;
+	rx_gatt_beacons();
+
+	struct bt_mesh_test_adv xmitparam = {
+		.retr = 5,
+		.interval = 20
+	};
+
+	/* Scan adv data. */
+	xmit_param = &xmitparam;
+	rx_xmit_adv();
+
+	/* Scan proxy beacons again. */
+	/* This is a workaround. If we set transmits to 5000 / 1000
+	 * the last beacon is sent +782 ms after the previous one.
+	 * Expected more than 1 sec. Zephyr's Controller bug?
+	 */
+	gattparam.transmits = 4000 / 1000;
```

Comment:
I guess this should be debugged.

Reply:
yep but not as part of this PR

Reply:
This is a Secure Network Beacon. If I disabled it through `CONFIG_BT_MESH_BEACON_ENABLE` and set transmit to `5000 / 1000`, then test passes. I guess the beacon can be disabled through Config Client.

Reply:
How did Secure Network Beacon pass Proxy Service parser? 
https://github.com/zephyrproject-rtos/zephyr/pull/50195/files#diff-27be331c9da5ebf00b50fdde2d30791a6fd470e79c5f65fa3fde138dc506a8a5R153

Reply:
It didn't, it affected time when proxy beacon was sent. I suggest you to try disabling SNB.